models.py

```diff
@@ -49,7 +49,7 @@ class Criterion(Deactivatable, db.Model):
     help_text = db.Column(db.String())
     adverse = db.Column(db.Boolean())
 
-    def __init__(self, category_id, title, recommendation_text, help_text, adverse):
+    def __init__(self, category_id=None, title=None, recommendation_text=None, help_text=None, adverse=None):
```

Comment:
This is interesting -- In my PR, I opted to make `category_id` required. I think this way of doing it makes sense as well, but we may want to add a validation that all criteria have a `category_id` on the model level. I have validation on the `services` level, which might not make sense. What do you think?

Reply:
I think making `category_id` required on the model level makes sense! I'll update that.

I agree that validations on the model level makes more sense – do we have any fields that are optional (other than `active` and `deactivated_at`?